pineappl_py/docs/source/modules/pineappl.rst

```diff
@@ -0,0 +1,42 @@
+pineappl package
```

Comment:
Replace this with `The Python API of PineAPPL`.

Reply:
done in 53fba7ba